doc/docs.md

```diff
@@ -4982,6 +4982,16 @@ executable, increasing your binary size, but making it more self contained
 and thus easier to distribute. In this case, `embedded_file.data()` will cause *no IO*,
 and it will always return the same data.
 
+`$embed_file` supports compression of the embedded file when compiling with `-prod`.
+
+```v ignore
+import os
+fn main() {
+	embedded_file := $embed_file('v.png', .zlib) // compressed using zlib
```

Comment:
One of the first things I can see someone asking is "What other compression types can I use?"